src/quill.mention.js

```diff
@@ -28,10 +28,15 @@ class Mention {
       offsetLeft: 0,
       isolateCharacter: false,
       fixMentionsToQuill: false,
+      defaultMenuOrientation: 'bottom',
     };
 
     Object.assign(this.options, options);
 
+    if (this.options.defaultMenuOrientation !== 'top' && this.options.defaultMenuOrientation !== 'bottom') {
+      this.options.defaultMenuOrientation = 'bottom';
```

Comment:
I added it in the case that someone accidentally passed in `null` or `sideways`, since the default in `this.options` would also be overridden by those values. Even still, `setMentionContainerPosition` only checks for the value `"top"`, so there's no need to force `defaultMenuOrientation` to either `"top"` or `"bottom"`. The intent was to limit the possible values of `this.options.defaultMenuOrientation` so it would be easier to develop against in the future.

All of that said, I don't mind getting rid of those lines, since they aren't necessary for the feature.